Tests/GitHubTrafficSupport.tests.ps1

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like this file is an accident?  Seems duplicative of the other test file.

Reply:
Yes, this is now deleted